### DISCUSSION TITLE
No emails after exiting AWS SES sandbox in eu-west-1

### DISCUSSION THREAD

**Discussion by osresearch** [mozilla/hubs#3119] [Hubs Cloud Questions] [answered]
I've deployed Hubs Cloud Personal in AWS' eu-west region and submitted a request to exit the AWS SES sandbox (as described [here](https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#youre-in-the-aws-sandbox-and-people-dont-receive-magic-link-emails)). AWS support replied:
>  Thank you for submitting your request to increase your sending limits. Your new sending quota is 50,000 messages per day. Your maximum send rate is now 14 messages per second. We have also moved your account out of the Amazon SES sandbox.
> This takes effect immediately in the Europe (Ireland) region. You can view the current sending rate and sending quota for your account on the Sending Statistics page of the Amazon SES console, or by using the GetSendQuota API.

However, the hubs admin console still thinks it is in the sandbox and attempting to log in does not generate any emails to any addresses other than to the admin account. Additionally, the SES console shows 0 emails sent during the past 24 hours and there is nothing in the recipient's spam folders.

During setup I did receive an email from AWS that mentions the us-east region:

> Congratulations! We successfully verified [your domain] in region US East (N. Virginia). You are now able to send email through Amazon SES and Amazon Pinpoint from any address within this domain. For more information, please refer our Developer Guide.

Is it possible to configure Hubs in the EU? I'd prefer to have it here for lower latency.

**Comment by robinkwilson** [comment, marked as answer]
Ah seems there was a mistake for the requested region for the SES increase. 
>  We have also moved your account out of the Amazon SES sandbox.
> This takes effect immediately in the Europe (Ireland) region. 

**Seems like you'll need to request the SES increase again in the US East (N. Virginia) region** since they unfortunately granted the increase in the Ireland region. I don't believe you'll have a problem getting a second increase in another region, others have done this too.

Because all of the SES is handled in the N. Virginia region, the earlier email likely verified your admin email address in that region.

**Comment by osresearch** [comment]
That fixed it, thanks!
